packages/react-scripts/template/src/service-worker-registration.js

```diff
@@ -0,0 +1,38 @@
+export default function register() {
```

Comment:
Can we put this in `react-dev-utils` and require (a) callback(s) to be passed for the results?
The less code we can burden the user with the better.